src/network/request.jsx

```diff
@@ -97,5 +97,5 @@ export default (options) => {
     throw networkError;
   };
 
-  return axios(requestConfig).then(pipeResponse).catch(rethrowError);
+  return axios(requestConfig).then(pipeResponse, rethrowError);
```

Comment:
Any argument for using this syntax? Because imho `Promise.catch()` is much more comprehensible.

Reply:
It actually does different things.
https://dmitripavlutin.com/javascript-promises-then-vs-then-catch/

Reply:
TIL